dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -489,9 +485,34 @@ else if ((cmd != null)
 		// *********************** END GRAZIANO issue-12-dnd-template		
 	}
 
+	/**
+	 * Returns true if a template is not being used by any html pages and can be deleted, false otherwise
+	 * @param template
+	 * @param user
+	 * @return true if template can be deleted
+	 * @throws LanguageException
+	 * @throws DotDataException
+	 * @throws DotSecurityException
+	 */
+	private boolean canTemplateBeDeleted (WebAsset template, User user, StringBuilder errorBuilder) throws LanguageException, DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 163 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)